build/build.sh

```diff
@@ -10,24 +10,49 @@ function removeFileIfExists {
     fi
 }
 
-# If there are less than six arguments passed to the build script (e.g. for local execution), try to set them automatically
+# Parsing arguments passed to the build script which should be key=value pairs
+for ARGUMENT in "$@"; do
+    KEY=$(echo $ARGUMENT | cut -f1 -d=)
+    KEY_LENGTH=${#KEY}
+    VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+    export "$KEY"="$VALUE"
+done
 
-if [[ $# -lt 1 ]]; then
-    echo "Error: No arguments passed. Make sure to pass at least the Netboot IP Address"
+# This argument is required to read the latest kernel version from the netboot server
+if [[ "$netbootIP" == "" ]]; then
+    echo "Error: No arguments passed. Make sure to pass at least the Netboot IP Address, e.g. netbootIP=10.1.30.4"
+    exit 1
 else
```

Comment:
This else is not really necessary.
With this else removed, the following if statements wouldn't be nested.